ccd-definition/ComplexTypes/GAStatementOfTruthGAspec.json

```diff
@@ -1,13 +1,13 @@
 [
   {
-    "ID": "StatementOfTruthGAspec",
+    "ID": "GAStatementOfTruthGAspec",
```

Comment:
Is there any specific reason for prefix GA?

Reply:
Yes.... As stated earlier in previous calls, the complex types get scanned lexicographically. So, in order to include it as part of the GeneralApplication complex type, we need to name it like this (since S comes after G)... It is the protocol that we have been following as discussed over calls in past.

Reply:
@smitatalawadekar-hmcts on a side note,  we need the relevant fields to be replicated in the civil-general-apps-ccd-definition after PR #27 for that repo gets merged in. Otherwise it may fall behind and would be harder to deal with in future. @faranhmcts please add comment as per necessary.